### PR TITLE
SEO-2245_Site_Audit_Issues

### PR DESCRIPTION
**Pull request by gowthamanthavasiyappan**
Hi @Aishwarya-Ganesan,

Worked on site audit issues, merge these changes.

Regards,
Gowthaman

### CODE REVIEW

File: Xamarin-iOS/SfAutoComplete/Overview.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 title: Overview of Syncfusion AutoComplete control for Xamarin.iOS
-description: Overview and key features of autocomplete control
+description: Learn here about Overview and key features of Syncfusion Xamarin.iOS AutoComplete Control, its elements, and more.
 platform: Xamarin.iOS
 control: AutoComplete
 documentation: ug
 ---
 
-# Overview
+# Overview of Xamarin.iOS AutoComplete
```
Comment (Aishwarya-Ganesan) [inline Xamarin-iOS/SfAutoComplete/Overview.md:10]: Xamarin.iOS AutoComplete Overview

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented

File: Xamarin-iOS/SfAutoComplete/Overview.md (hunk 1 above)
Comment (gowthamanthavasiyappan) [reply Xamarin-iOS/SfAutoComplete/Overview.md]: Changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine

**Pull request merged**